mqtt/policy/src/core/mod.rs

```diff
@@ -843,4 +833,98 @@ pub(crate) mod tests {
             policy.starts_with(input)
         }
     }
+
+    #[cfg(feature = "proptest")]
+    mod proptests {
```

Comment:
Do you want to split proptest generators and test themselves and move generators to a separate file?

Reply:
Good suggestion, but at the moment I don't see a strong reason to do that.